### PR TITLE
Updated help.py plugin

### PR DESCRIPTION
**Pull request by Lbatson**
changed help with no args to display list and info on single line. strip new lines from descriptions to present info in no more than two lines. one for short_desc and one for long_desc if present.

Example: 
screen shot 2013-05-10 at 3 02 05 pm

### CODE REVIEW

File: hamper/plugins/help.py [hunk 1]
```diff
@@ -62,8 +58,8 @@ def command(self, bot, comm, groups):
                 return
 
             if command.short_desc:
-                bot.reply(comm, '{0.short_desc}'.format(command))
+                bot.reply(comm, '{0.short_desc}'.format(command).replace('\n', ' '))
```
Comment (chancez) [inline hamper/plugins/help.py:61]: Not sure if this is something that I really like. It really takes away from the readability of help

File: hamper/plugins/help.py [hunk 2]
```diff
@@ -34,18 +34,14 @@ class HelpMainMenu(Command):
...
         short_desc = 'help [command] - Show help for commands.'
-        long_desc = ('For detailed help on a command, say "!help command". '
+        long_desc = ('For detailed help on a command, say "!help <command name>". '
                       'Some commands may not be listed. If you think they '
                       'should, poke the plugin author.')
 
         def command(self, bot, comm, groups):
             commands = Help.helpful_commands(bot)
-            response = ['Available commands']
-            for command in commands:
```
Comment (chancez) [inline hamper/plugins/help.py:44]: Can we move the parts removed below this (the old implementation) into a new command? !help brief is probably good.

Comment (Lbatson) [inline hamper/plugins/help.py:44]: So what would help with no args do?

File: hamper/plugins/help.py (hunk 1 above)
Comment (Lbatson) [inline hamper/plugins/help.py:61]: So you prefer having it print out multiple lines? The whole point of the commit was to condense the messages. I don't see where it retracts from readability. The first line output is a short description. If a long description is available it prints the whole thing. Seems pretty straightforward

Comment (chancez) [inline hamper/plugins/help.py:61]: I feel like that this is more about fixing the default functionality from printing 10 lines of help, not really condensing help.

File: hamper/plugins/help.py (hunk 2 above)
Comment (chancez) [inline hamper/plugins/help.py:44]: Help with no commands is exactly what lines 43 and 44 are doing in this commit. You've already done that.

Comment (Lbatson) [inline hamper/plugins/help.py:44]: Oh i see, you're talking about moving the removed lines to a separate function or new command where it prints the list with their short descriptions? I assume we'll just string them along in one message as opposed to printing out a line for each one?

File: hamper/plugins/help.py (hunk 1 above)
Comment (Lbatson) [inline hamper/plugins/help.py:61]: Yea exactly, removing the new lines condensed the amount of lines. That was the goal correct?

**Comment by chancez** [comment]
This is quite out of date and not quite as relevant any more so im going to close this.

**Pull request closed without merging**